src/components/chips/contact-chips.spec.js

```diff
@@ -0,0 +1,104 @@
+describe('<md-contact-chips>', function() {
+  var scope;
+  var CONTACT_CHIPS_TEMPLATE = '\
+      <md-contact-chips\
+          ng-model="contacts"\
+          md-contacts="querySearch($query)"\
+          md-contact-name="name"\
+          md-contact-image="image"\
```

Comment:
This is generating a 404 in the Karma Webserver.
Can you please do hardcoded data image in the ng-src; @see https://css-tricks.com/data-uris/

e.g
```
data:image/gif;base64,R0lGODlhEAAQAMQAAORHHOVSKudfOulrSOp3WOyDZu6QdvCchPGolfO0o/XBs/fNwfjZ0frl3/zy7////wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAkAABAALAAAAAAQABAAAAVVICSOZGlCQAosJ6mu7fiyZeKqNKToQGDsM8hBADgUXoGAiqhSvp5QAnQKGIgUhwFUYLCVDFCrKUE1lBavAViFIDlTImbKC5Gm2hB0SlBCBMQiB0UjIQA7
```